seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -5202,6 +5202,7 @@
       ],
       "tests": [
         "assert(code.match(/for\\s*\\(/g).length > 1, 'message: You should be using a <code>for</code> loop for this.');",
+        "assert(code.match(/push\\s*\\(/g).length > 1, 'message: You should be using <code>push</code> like the example.');",
```

Comment:
~~'message: You should be using <code>push</code> like the example.'~~

'message: You should use the array method <code>push</code>.'